### PR TITLE
Known Issue: MariaDB Check and Repair

### PR DESCRIPTION
**Pull request by nemchik**
Ref: https://github.com/linuxserver/docker-mariadb/pull/124

### CODE REVIEW

File: content/issues/2023-05-28-mariadb.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+title: 'Known Issue: MariaDB Check and Repair'
+date: '2023-05-28 12:00:00Z'
+severity: notice
+resolved: false
+affected:
+  - 'Known Issues'
+  - 'mariadb'
+section: 'issue'
+---
+There is a known issue with MariaDB when updating the container image and a database upgrade is required on user databases that are not in a healthy state. This can prevent MariaDB from starting.
```
Comment (thespad) [inline content/issues/2023-05-28-mariadb.md:11]: I would rephrase this to something like:
```
There is a known issue with MariaDB when updating the container image to a new version. If a database upgrade is required and user databases are not in a healthy state, then the MariaDB service may fail to start.
```

File: content/issues/2023-05-28-mariadb.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+MariaDB version updates (ex: `10.6.x` to `10.11.x`) can happen when updating the container image. Upgrading user databases is done manually by running a command.
+
+When the container initializes, if `MYSQL_ROOT_PASSWORD` is set an upgrade check will run. If an upgrade is required the log will indicate the need to run:
```
Comment (thespad) [inline content/issues/2023-05-28-mariadb.md:15]: This needs a note about making sure anything using the database is stopped/disconnected before running the upgrade or check commands.

File: content/issues/2023-05-28-mariadb.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+```log
+Caught SIGTERM signal!
+```
+
+- Roll back the container image to a previous tag (ex: `lscr.io/linuxserver/mariadb:10.6.13` assuming this was the version you were running when things were last working)
+- Run the check, repair, analyze, and optimize commands above, watch for any issues
+- Update the container image to the new tag (ex: `lscr.io/linuxserver/mariadb:10.11.3` or `lscr.io/linuxserver/mariadb:latest`)
+- Run the upgrade command above
+
+This should get MariaDB running again with your user databases in a healthy state.
```
Comment (thespad) [inline content/issues/2023-05-28-mariadb.md:43]: I would add a note at the end to the effect of
```
If running MariaDB databases in production, we always recommend pinning the container to a specific version tag, such as `version-10.6.13-r0`, and planning any upgrades so that you can test and monitor them.
```

Comment (nemchik) [reply content/issues/2023-05-28-mariadb.md]: Resolved in 5966df5b9120f16eeee8ba1379e7ec691c1a6197

File: content/issues/2023-05-28-mariadb.md (hunk 2 above)
Comment (nemchik) [reply content/issues/2023-05-28-mariadb.md]: Resolved in 5966df5b9120f16eeee8ba1379e7ec691c1a6197 also applied to https://github.com/linuxserver/docker-mariadb/pull/124 in https://github.com/linuxserver/docker-mariadb/pull/124/commits/bef18099cfbf718aa66d78cce629e1f4ffc9ce8c

File: content/issues/2023-05-28-mariadb.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: 'Known Issue: MariaDB Check and Repair'
+date: '2023-05-28 12:00:00Z'
```
Comment (thespad) [inline content/issues/2023-05-28-mariadb.md:3]: Update the publish date - if it's more than 24 hours ago the RSS bot won't pick it up automatically.
